lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -116,6 +117,7 @@ class ConnectedItineraryBody extends Component {
       <ItineraryBodyContainer>
         <StyledItineraryBody
           accessibilityScoreGradationMap={accessibilityScoreGradationMap}
+          alwaysCollapseAlerts={allowUserAlertCollapsing}
```

Comment:
```suggestion
          alwaysCollapseAlerts={!allowUserAlertCollapsing}
```
Should this be flipped?

Reply:
I don't think so, I'm setting `allowUserAlertCollapsing` to `true` in the config!

Reply:
Unless we want alerts to collapse by default, which might be preferable?

Reply:
But as it is right now if `allowUserAlertCollapsing` is true, then they won't be allowed to collapse them right? Or am I forgetting how the behavior works

Reply:
No, `alwaysCollapseAlerts` needs to be true for the alerts to collapse